### DISCUSSION TITLE
Formatting Errors when uploading pubmed abstracts into ASReview

### DISCUSSION THREAD

**Discussion by metaregressionman** [asreview/asreview#1287] [Troubleshooting (report errors in the "Issues" tracker)]
Hello all,

I keep getting the error "expected 2 fields in line 6830, saw 3" when trying to upload Pubmed abstracts into ASReview. Does anyone know how to get around this? Thank you for any help!

**Comment by Rensvandeschoot** [comment]
Is it the same issue as reported in https://github.com/asreview/asreview/issues/871? A solution is offered in that thread.
